root/etc/systemd/system/network.service.d/network-recovery-cleanup.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExecStartPre=-/usr/sbin/ip link delete brtmp
```

Comment:
The ExecStartPre should not generate output. In case of error, where is redirect the stderr?

We should also test carefully this scenario:
- run the network recovery tool
- create a new bridge from the UI
- check the interface-update event doesn't raise any error